pkg/auth/handler/webapp/verify_identity.go

```diff
@@ -175,6 +176,7 @@ func (h *VerifyIdentityHandler) ServeHTTP(w http.ResponseWriter, r *http.Request
 		}
 
 		code := r.Form.Get("x_verification_code")
+		code = strings.TrimSpace(code)
```

Comment:
Instead of trimming the space here, find out where is the code verified and modify the verification logic.

Some hints on how to locate the places:
- We are dealing with 6-digit OTP so which `"format"` correspond to 6-digit OTP?
- Where are the corresponding `"format"` being used?
- 6-digit OTP is being in verification, primary authentication and secondary authentication.